pkg/publish.go

```diff
@@ -28,6 +20,11 @@ func Publish(file string, rootID string, dryRun bool) (id string, err error) {
 		return id, errors.New("space must be set in config")
 	}
 
+	repoBrowseURL := viper.GetString("repo_browse_url")
```

Comment:
What do you think about `browse_url_base`? Or perhaps even better would be to allow users to specify a format string?

```
browse_uri: https://github.com/jesselang/dox/blame/master # which would append "/%s" if not present in the string
# which would allow extra parameters to be passed
browse_url: https://othersourcesite.com/repo-base/browse/%s?format=raw
```